packages/platform-browser/animations/src/module.ts

```diff
@@ -83,6 +83,7 @@ export class BrowserAnimationsModule {
 export function provideAnimations(): Provider[] {
   // Return a copy to prevent changes to the original array in case any in-place
   // alterations are performed to the `provideAnimations` call results in app code.
+  performanceMarkFeature('NgEagerAnimations');
```

Comment:
Let's put the call before comments, since they describe the next line where we clone an array.
```suggestion
  performanceMarkFeature('NgEagerAnimations');
  // Return a copy to prevent changes to the original array in case any in-place
  // alterations are performed to the `provideAnimations` call results in app code.
```